README.md

```diff
@@ -3,6 +3,7 @@
 Jamm provides `MemoryMeter`, a Java agent for all Java versions to
 measure actual object memory use including JVM overhead.
 
+Jamm assume that the JVM running the code is an HotSpot JVM. It has not been tested with other type of JVMs.
```

Comment:
I do not think that we should try to support Eclipse OpenJ9 at least at this stage. That would probably mean a lot of work.

Reply:
My concern was more that we already opted into that with the previous jamm version but I guess we should be explicit that this is not the case in the new version, one more breaking change... Otherwise, I totally agree that it is a rabbit hole :(

Reply:
I added a comment about it in the README